simpledotorg/simple-server
# 3944 imo notification action

**Story card:** [ch3944](https://app.clubhouse.io/simpledotorg/story/3944/add-ability-to-send-imo-messages)

## Because

We need the ability to send notifications through Imo.

## Test instructions

If you want to get wild, you can install Imo on your phone, change a patient record to have your phone number, add the imo password from 1pass, and send yourself a notification through this class. I've done it locally and don't expect a reviewer to do that.
